upstream-community-operators/kubevirt/kubevirtoperator.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: kubevirt
+channels:
+  - name: beta
```

Comment:
Side note - Let#s keep this channel as beta until we reach KubeVirt v1

Reply:
@fabiand set this to alpha based on you comments below.

Reply:
Yes!